configure.ac

```diff
@@ -303,6 +303,22 @@ AM_CONDITIONAL([WITH_OPENSSL], [test x$with_openssl != xno])
 AC_DEFINE_UNQUOTED([_ODP_OPENSSL], [$have_openssl],
 	  [Define to 1 to enable OpenSSL support])
 
+##########################################################################
+# Configure/disable usage of ARMV8 Crypto library for AES-GCM and AES-CBC
+##########################################################################
+AC_ARG_WITH([armv8crypto],
+	    [AS_HELP_STRING([--with-armv8crypto],
+			    [compile with armv8 crypto.Should be used along with --without-openssl]
+			    [[default=without]])],
+	    [with_armv8crypto=yes],
+	    [])
+AS_IF([test "$with_armv8crypto" == "yes"],
+      [ODP_ARMV8_CRYPTO
+       have_armv8crypto=1], [have_armv8crypto=0])
+AM_CONDITIONAL([WITH_ARMV8_CRYPTO], [test x$with_armv8crypto == xyes])
+AC_DEFINE_UNQUOTED([_ODP_ARMV8_CRYPTO], [$have_armv8crypto],
+	  [Define to 1 to enable armv8 crypto support])
+
```

Comment:
I think it would be better to let users select the crypto implementation with a new configure option in the same manner as the default scheduler can be selected currently.
E.g.
```
--enable-crypto-default  choose default crypto implementation [openssl/armv8crypto/null] (linux-generic)
```
You can check `platform/linux-generic/m4/odp_scheduler.m4` for an example. Since this would be a platform specific option, it should go to for example `platform/linux-generic/m4/odp_armv8crypto.m4`

Reply:
There should also be a configuration option for pointing to the AArch64cryptolib installation directory.
E.g.
```
  --with-aarch64crypto-path=DIR    path to AArch64cryptolib install directory [default=system] (linux-generic)
```

Reply:
Configure phase should also check that the library is available. `m4/odp_openssl.m4`is probably a good example.